addons_updater/rootfs/scripts/00-aaa_dockerfile_backup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/with-contenv bashio
```

Comment:
Hi, you can't use bashio here - the point is to install it if the dockerfile build did not work. It is needed in context such as lock builds on supervised installations where curl doesn't seem to work. This was tested with several users of my addon and this was the only working solution to allow local builds on supervised installs. But then it requires #!/bin/bash

Reply:
that is not good, sorry will change it all asap!